src/pages/instant-observability.js

```diff
@@ -660,38 +647,25 @@ const QuickstartsPage = ({ data, location }) => {
           <div
             css={css`
               padding: 1.25rem 0;
-              font-size: 0.9rem;
-              color: var(--secondary-text-color);
+              font-size: 16px;
+              color: var(--color-neutrals-800);
               display: flex;
               justify-content: space-between;
+              align-text: center;
+
+              strong {
+                color: var(--color-neutrals-700);
+              }
             `}
           >
-            <span>Showing {filteredQuickstarts.length} results</span>
-
-            <div
+            <span
               css={css`
-                min-width: 155px;
-                margin-left: 20px;
-                display: inline;
-
-                @media screen and (max-width: 1180px) {
-                  margin-left: 0px;
-                }
+                color: var(--primary-text-color);
```

Comment:
We need to add a text color for darkmode, the current color blends in with the background:
<img width="445" alt="Screen Shot 2022-01-03 at 2 15 34 PM" src="https://user-images.githubusercontent.com/70179303/147986333-cf15fb57-f32d-4923-85f5-3b5046c3a76d.png">